### DISCUSSION TITLE
Difference between Scribble and filament-tiptap-editor?

### DISCUSSION THREAD

**Discussion by thijskuilman** [awcodes/scribble#68] [Q&A] [answered]
I'm a bit confused about Scribble compared to the [filament-tiptap-editor](https://github.com/awcodes/filament-tiptap-editor), since they seem really similar. Is Scribble supposed to be the replacement of the filament-tiptap-editor down the road? Or is there a difference in vision between those two?

I am considering to implement one of these packages in my project, so I would love an answer to this!

**Comment by awcodes** [comment, marked as answer]
Fair question. lol.

Scribble is the next major release of tiptap editor plugin. However, Scribble will be getting ported to another version that uses Filament Actions instead of Svelte for the editor functionality. But basically they are all the same thing.
